Sources/Xcodeproj/pbxproj().swift

```diff
@@ -431,7 +431,7 @@ func xcodeProject(
                 targetSettings.common.SWIFT_FORCE_STATIC_LINK_STDLIB = "NO"
                 targetSettings.common.SWIFT_FORCE_DYNAMIC_LINK_STDLIB = "YES"
 
-                targetSettings.common.LD_RUNPATH_SEARCH_PATHS += ["@executable_path"]
+                targetSettings.common.LD_RUNPATH_SEARCH_PATHS += ["@executable_path", "$(inherited)"]
```

Comment:
This is not required I think. It will add the `$(inherited)` again.

Reply:
Good catch, didn't see the "+" there